media-api/app/lib/querysyntax/QuerySyntax.scala

```diff
@@ -178,6 +191,20 @@ class QuerySyntax(val input: ParserInput) extends Parser with ImageFields {
     })
   }
 
+  def MatchMimeTypeValue = rule {
+    (QuotedString | String) ~> parseMimeType _
+  }
+
+  def parseMimeType(expr: String): Value = {
```

Comment:
Could we make use of the [`MimeType` trait](https://github.com/guardian/grid/blob/e7d1d7f9f8263039c831b3c0c9255c6f0d318952/common-lib/src/main/scala/com/gu/mediaservice/model/MimeType.scala#L8) here to increase type safety?

Reply:
Great idea. I also implemented a rule for just accepting the formats.